site/content/docs/main/data-movement-node-selection.md

```diff
@@ -153,7 +153,7 @@ The other VGDP instances will run on nodes, which instance type is `Standard_B4m
 
 Velero data mover supports to use different StorageClass to create backupPVC by [design](https://github.com/vmware-tanzu/velero/pull/7982).
 
-In this example, if the backup target PVC's StorageClass is `kibishii-storage-class`, its backupPVC should use StorageClass `worker-storagepolicy`. Because the final StorageClass is `worker-storagepolicy`, the backupPod uses the loadAffinity specified by `loadAffinity`'s elements with `StorageClass` field set to `worker-storagepolicy`. backupPod will be assigned to nodes, which instance type is `Standard_B2ms`.
+**Warning**: Due to the current limitation where only the first element in the `loadAffinity` array is processed, only the first selector (nodes with instance type `Standard_B4ms` and storage class `kibishii-storage-class`) will be applied. The second element will be ignored. To work around this limitation, you would need to combine all necessary conditions into a single loadAffinity element.
```

Comment:
I don't think we need this one, for each of the example, as we have already clarified it as an overall statement.

Reply:
```suggestion
In this example, if the backup target PVC's StorageClass is `kibishii-storage-class`, its backupPVC should use StorageClass `worker-storagepolicy`. Because the final StorageClass is `worker-storagepolicy`, the backupPod uses the loadAffinity specified by `loadAffinity`'s elements with `StorageClass` field set to `worker-storagepolicy`. backupPod will be assigned to nodes, which instance type is `Standard_B2ms`.
```